test/presenters/notifications/setup/review.presenter.test.js

```diff
@@ -0,0 +1,253 @@
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const RecipientsFixture = require('../../../fixtures/recipients.fixtures.js')
+
+// Thing under test
+const ReviewPresenter = require('../../../../app/presenters/notifications/setup/review.presenter.js')
+
+describe('Notifications Setup - Review presenter', () => {
+  let testRecipients
+  let testInput
+
+  beforeEach(() => {
+    testRecipients = RecipientsFixture.recipients()
+    testInput = Object.values(testRecipients)
+  })
+
+  describe('when provided with "recipients"', () => {
+    it('correctly presents the data', () => {
+      const result = ReviewPresenter.go(testInput)
+
+      expect(result).to.equal({
+        pageTitle: 'Send returns invitations',
+        recipients: [
+          {
+            licences: [`${testRecipients.primaryUser.all_licences}`],
+            method: 'Email - primary user',
+            contact: ['primary.user@important.com']
+          },
+          {
+            licences: [`${testRecipients.returnsAgent.all_licences}`],
+            method: 'Email - returns agent',
+            contact: ['returns.agent@important.com']
+          },
+          {
+            licences: [`${testRecipients.licenceHolder.all_licences}`],
+            method: 'Letter - licence holder',
+            contact: [
+              'Licence Guy',
+              'undefined',
+              'Privet Drive',
+              'Surrey',
+              'Harry',
+              'J',
+              'WD25 7LR',
+              'Little Whinging',
+              'Person',
+              'Licence holder'
+            ]
+          },
+          {
+            licences: [`${testRecipients.returnsTo.all_licences}`],
+            method: 'Letter - Returns To',
+            contact: [
+              'Returner Guy',
+              'undefined',
+              'Privet Drive',
+              'Surrey',
+              'Harry',
+              'J',
+              'WD25 7LR',
+              'Little Whinging',
+              'Person',
+              'Returns to'
+            ]
+          },
+          {
+            licences: testRecipients.licenceHolderWithMultipleLicences.all_licences.split(','),
+            method: 'Letter - licence holder',
+            contact: [
+              'Multiple Licence Guy',
+              'undefined',
+              'Privet Drive',
+              'Surrey',
+              'Harry',
+              'J',
+              'WD25 7LR',
+              'Little Whinging',
+              'Person',
+              'Licence holder'
+            ]
+          }
+        ],
+        recipientsAmount: 5
+      })
+    })
+
+    describe('the "pageTitle" property', () => {
+      it('should return page title', () => {
+        const result = ReviewPresenter.go(testInput)
+        expect(result.pageTitle).to.equal('Send returns invitations')
+      })
+    })
```

Comment:
The intent of adding the `correctly presents the data` test was that it means we can focus on only those properties with branching/complex logic in the subsequent tests.

So, we can drop this test and any others like it.

```suggestion
```